### DISCUSSION TITLE
How to Fix Video Lag in Microsoft Teams on Windows

### DISCUSSION THREAD

**Discussion by toshikijp** [andatoshiki/toshiki-discuss#32] [Q&A] [answered]
As the question describes

**Comment by andatoshiki** [comment, marked as answer]
Idiotic question, proceed the steps as follows to quick troubleshoot your root issue for video lag in MS Teams
1. Check Your Internet Connection. ...
2. Use the Call Health View to Monitor Call and Meeting. ...
3. Check for Microsoft Teams and Windows Update. ...
4. Update Your Intel Graphic Drivers. ...
5. Run Microsoft Teams Network Assessment Tool. ...
6. Reinstall Microsoft Teams.
